mayastor/src/bin/mayastor.rs

```diff
@@ -24,29 +24,24 @@ fn start_tokio_runtime(args: &MayastorCliArgs) {
     let endpoint = args.mbus_endpoint.clone();
 
     Mthread::spawn_unaffinitized(move || {
-        let rt = tokio::runtime::Builder::new_multi_thread()
-            .worker_threads(4)
-            .max_blocking_threads(4)
-            .on_thread_start(Mthread::unaffinitize)
-            .enable_all()
-            .build()
-            .unwrap();
-        let _r = rt.enter();
-        if let Some(endpoint) = endpoint {
-            debug!("mayastor mbus subsystem init");
-            message_bus_init();
-            mbus_api::message_bus_init_tokio(endpoint);
-            Registration::init(&node_name, &grpc_address.to_string());
-        }
+        runtime::block_on(async move {
+            let mut futures = Vec::new();
+            if let Some(endpoint) = endpoint {
+                debug!("mayastor mbus subsystem init");
+                message_bus_init();
```

Comment:
I meant to ask you about this before, do we need to call these message bus inits? IIRC the registration does the init should do the message bus stuff for us.

Reply:
I don't know, to be honest, that code has 3 different init functions and is rather confusing, to be honest. I don't understand why it cants simply be one future that does all the work and be done with it :)